terraform/deploy_new_dashboard_ami.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+
+# deploy_new_database_ami.sh region workspace_name
```

Comment:
I'm late to the review game here, but `deploy_new_database_ami` -> `deploy_new_dashboard_ami`

Reply:
Ack, I should've done a find/replace. The mistakes we make trying to get something working in a hurry.